modules/ROOT/pages/apikit-customize-soap-fault-msg.adoc

```diff
@@ -136,11 +136,11 @@ The application should deploy, and the following messages should appear in the C
 
 == Send a SoapUI Request
 
-. Using SoapUI, send a request to the deployed Mule app on port 8087.
+. Using SoapUI, send a request to the deployed the Mule app on port 8087.
```

Comment:
There is an extraneous "the" before Mule app here

Reply:
Thanks!